cmake/projects/eyalroz_printf/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 Alexander Voronov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_report_broken_package)
+include(hunter_status_debug)
+
+hunter_add_version(
+    PACKAGE_NAME eyalroz_printf
+    VERSION "6.2.0-b1"
```

Comment:
What's up with the -b1 art of the version number?

Reply:
Hi, @eyalroz !
Tell us why you used the `-b1` prefix for package version?

Reply:
I didn't create that version string... but - version 6.2.0 is not released yet, it's just in beta. The tag is: "v6.2.0-b1". There are a few more commits since that tag.

Reply:
Ok got it. Then we'll wait for your release and then I'll correct the version and related things. 
In the meantime, I'll improve the documentation in .rst.

Reply:
@eyalroz thanks for taking the time and answering. Very kind of you

@crvux do you need a feature from the beta release? Or would it be enough for you to use the latest non-beta tag for now and once `6.2.0` is released add that version to hunter?